lib/Jsrt/JsrtDiag.cpp

```diff
@@ -269,17 +269,12 @@ CHAKRA_API JsDiagGetBreakpoints(
 #ifndef ENABLE_SCRIPT_DEBUGGING
     return JsErrorCategoryUsage;
 #else
-    return GlobalAPIWrapper_NoRecord([&]() -> JsErrorCode {
-
+    return ContextAPIWrapper_NoRecord<false>([&](Js::ScriptContext* scriptContext) -> JsErrorCode {
         PARAM_NOT_NULL(breakpoints);
-
         *breakpoints = JS_INVALID_REFERENCE;
 
-        JsrtContext *currentContext = JsrtContext::GetCurrent();
-
-        Js::JavascriptArray* bpsArray = currentContext->GetScriptContext()->GetLibrary()->CreateArray();
-
-        JsrtRuntime * runtime = currentContext->GetRuntime();
+        JsrtContext* currentContext = JsrtContext::GetCurrent();
+        JsrtRuntime* runtime = currentContext->GetRuntime();
```

Comment:
Should this return JsErrorNoCurrentContext if currentContext is nullptr?

Reply:
ContextAPIWrapper handles that check for us.

Reply:
But you dereference currentContext to call GetRuntime();

Reply:
This whole body is contained within a lambda that's passed to `ContextAPIWrapper`.  It will only call the lambda if it first verifies that there's a context:

https://github.com/Microsoft/ChakraCore/blob/76373f24141ee66bc5e9e4ff131f097f96542a95/lib/Jsrt/JsrtInternal.h#L174-L182